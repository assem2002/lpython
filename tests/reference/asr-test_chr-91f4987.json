{
    "basename": "asr-test_chr-91f4987",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_chr.py",
    "infile_hash": "6520b636dd89e03d59caf9be2e7d6ec001e758ec0a92de2286b691bc",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_chr-91f4987.stderr",
    "stderr_hash": "442cc31382f37e87185b3e3ec3f86831ce9f7a64af3a6cd2655d5c7b",
    "returncode": 2
}