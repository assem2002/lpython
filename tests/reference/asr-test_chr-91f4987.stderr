semantic error: 'f64' object cannot be interpreted as an integer
 --> tests/errors/test_chr.py:3:13
  |
3 |     s = chr(973.2)
  |             ^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
